src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1636,7 +1631,7 @@ private function createVersion(ContainerBuilder $container, ?string $version, ?s
         return new Reference('assets.empty_version_strategy');
     }
 
-    private function registerTranslatorConfiguration(array $config, ContainerBuilder $container, LoaderInterface $loader, string $defaultLocale, array $enabledLocales): void
+    private function registerTranslatorConfiguration(array $config, ContainerBuilder $container, LoaderInterface $loader, string $defaultLocale): void
```

Comment:
For the translator, there is only this type to modify to accept a parameter. The `default.translator` service car receive an `%env(...)%` value that is resolved by the DIC at runtime.

```suggestion
    private function registerTranslatorConfiguration(array $config, ContainerBuilder $container, LoaderInterface $loader, string $defaultLocale, array|string $enabledLocales): void
```